html/js/app.js

```diff
@@ -448,6 +448,8 @@ QB.Phone.Functions.LoadPhoneData = function(data) {
     QB.Phone.Functions.LoadMetaData(data.PhoneData.MetaData);
     QB.Phone.Functions.LoadContacts(data.PhoneData.Contacts);
     QB.Phone.Functions.SetupApplications(data);
+    $("#phone-serverid").html("<span style='font-size: 1.3vh; font-weight:bold;'>" + data.serverid + "</span>");
+    console.log("Phone succesfully loaded!");
```

Comment:
I think this was removed in the most recent updates? It's a debug print.. doesn't really serve a purpose to anyone.